### DISCUSSION TITLE
We have a meeting time! - Friday, 11/10/2024, 4Pm UTC

### DISCUSSION THREAD

**Discussion by Rayccheal** [clentafrica/hr-people-operations#11] [Announcements]
Dear Team Leads, @clentafrica/wg-design @clentafrica/wg-disability-gender-equality @clentafrica/wg-education @clentafrica/wg-energy @clentafrica/wg-marketing-comms @clentafrica/wg-strategy @clentafrica/wg-monitoring-and-evaluation @clentafrica/wg-web-dev @clentafrica/wg-fundraising @clentafrica/wg-sustainability-esg @clentafrica/wg-hr @clentafrica/wg-gsod @clentafrica/wg-health @clentafrica/wg-community-managers @clentafrica/wg-research-and-data @clentafrica/wg-programs @clentafrica/wg-gsoc @Onodjayeke @yigakpoa 

I would like to take a moment to sincerely appreciate each of you for your dedication and enthusiasm for our common goals.  It is incredibly motivating!

I’m excited to announce that our trailblazing Team Leads meeting will take place tomorrow, Friday, October 11, 2024, at 4 PM UTC.

Please check with your local time zones [here ](https://www.worldtimebuddy.com/?pl=1&lid=100,121,30,5,8&h=100&hf=1) for confirmation. 

**Meeting Time**: 4Pm UTC
**Day**: Friday, 11th October, 2024
**Meeting Link**: https://us05web.zoom.us/j/84369427414?pwd=YK4dSaro36eMcR3F4fgR6LzurSDMTi.1

See you there!

Regards,
Racheal

**Comment by Rayccheal** [comment]
Hello Team Leads, @clentafrica/wg-design @clentafrica/wg-health @clentafrica/wg-disability-gender-equality @clentafrica/wg-hr @clentafrica/wg-blog @clentafrica/wg-marketing-comms @clentafrica/wg-web-dev @clentafrica/wg-strategy @clentafrica/wg-education @clentafrica/wg-energy @clentafrica/wg-sustainability-esg @clentafrica/wg-research-and-data @clentafrica/wg-fundraising @clentafrica/wg-monitoring-and-evaluation @clentafrica/wg-community-managers @clentafrica/wg-socialmedia @clentafrica/wg-programs @yigakpoa @Onodjayeke 

**Please join the meeting using this link**: https://us05web.zoom.us/j/84369427414?pwd=YK4dSaro36eMcR3F4fgR6LzurSDMTi.1

PS: Meeting starts in 20 minutes!

See you soon!

Racheal.

**Comment by aramide-dess** [comment]
Hi! Trying to join the meeting and it says the host has another meeting in progress.

**Reply by Rayccheal** [reply]
I'm so sorry about that! Please join with this https://us05web.zoom.us/j/89503489094?pwd=UDBvX95c0mtBEMHK8v8u0tW1KdZmxg.1

**Comment by MadhalasaSJ** [comment]
Hi, in zoom it is shown that the meeting is tomorrow. Can anyone confirm when's the meeting is schedule

**Reply by Rayccheal** [reply]
https://us05web.zoom.us/j/89503489094?pwd=UDBvX95c0mtBEMHK8v8u0tW1KdZmxg.1 Joi  with this! 
 I am so sorry

**Reply by adilkadivala** [reply]
this is also not working

**Reply by yigakpoa** [reply]
https://us05web.zoom.us/j/89503489094?pwd=UDBvX95c0mtBEMHK8v8u0tW1KdZmxg.1

**Comment by Abraham100000** [comment]
Same here.

**Reply by Rayccheal** [reply]
https://us05web.zoom.us/j/89503489094?pwd=UDBvX95c0mtBEMHK8v8u0tW1KdZmxg.1

**Reply by yigakpoa** [reply]
@Abraham100000 please use this: https://us05web.zoom.us/j/89503489094?pwd=UDBvX95c0mtBEMHK8v8u0tW1KdZmxg.1

**Comment by Rayccheal** [comment]
Let's join with this link please: https://us05web.zoom.us/j/89503489094?pwd=UDBvX95c0mtBEMHK8v8u0tW1KdZmxg.1

The first one is not working, I am so sorry

**Comment by adilkadivala** [comment]
provided link not working

**Reply by yigakpoa** [reply]
@adilkadivala please use this: https://us05web.zoom.us/j/89503489094?pwd=UDBvX95c0mtBEMHK8v8u0tW1KdZmxg.1

**Reply by adilkadivala** [reply]
it is asking for Launch Meeting . 

not get any option to join

provided link redirects here

![join](https://github.com/user-attachments/assets/e9183dac-5d4e-4409-be7c-f3d839bbdf01)

**Reply by yigakpoa** [reply]
Yeah can you launch the meeting and see what it says?

**Reply by adilkadivala** [reply]
please share roomId and password,  it will easy to join in desktop app

**Reply by adilkadivala** [reply]
![share](https://github.com/user-attachments/assets/e4c7dc8f-bbd0-440a-99db-2079ffc4d7f5)

**Comment by yigakpoa** [comment]
@Rayccheal it says waiting for you to start the meeting

**Comment by Faithmalokotsa** [comment]
I'll join in
